ui/apps/platform/src/Components/SubHeader/SubHeader.js

```diff
@@ -1,17 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const SubHeader = ({ text, capitalize }) => {
-    return <div className={`mt-1 ${capitalize ? 'capitalize' : ''}`}>{text}</div>;
```

Comment:
Don't we need to keep the `mt-1` TW classname for now, for spacing?

Reply:
It looks unnecessary for 2 reasons:
* Parent `div` has `self-center` class, so classic `pt-1` for `h1` element shifted it off center.
* Because both `h1` and `div` have PatternFly default `line-spacing: 1.5` this margin looked unnecessary and inconsistent with wrapped elements in table cells.

![self-center](https://github.com/stackrox/stackrox/assets/11862657/71b03cd0-cf92-4ce0-8bdb-8b0db24f706f)